AppDevAnnouncements/Classes/NotificationViewController.swift

```diff
@@ -8,26 +8,21 @@
 
 import UIKit
 
-public protocol NotificationDelegate: class {
-    func present(_ notificationViewController: NotificationViewController)
-}
-
-public class NotificationViewController: UIViewController {
+fileprivate class NotificationViewController: UIViewController {
 
     /// Components
     private var notificationView: NotificationView!
 
     /// Initializer variables
     private var announcement: Announcement
-    private weak var delegate : NotificationDelegate?
 
-    public init(announcement: Announcement, delegate: NotificationDelegate) {
+    fileprivate init(announcement: Announcement) {
         self.announcement = announcement
-        self.delegate = delegate
         super.init(nibName: nil, bundle: nil)
+        self.modalPresentationStyle = .overFullScreen // Will eventually be changed to a custom presentation animation
     }
 
-    override public func viewDidLoad() {
+    override fileprivate func viewDidLoad() {
```

Comment:
Does this need to be file private if the class already is? I would cut unnecessary access modifiers.